tests/performance-test/performance-test.sh

```diff
@@ -35,17 +35,32 @@ ENDUSAGE
     exit 1
 }
 
+ELIPSE="."
```

Comment:
Can you add some docs on what this function in providing? I see that it changes one type of ellipses (note the spelling, should be 2 L's) to another, but it's not exactly clear at this point why this is necessary.

Reply:
For sure. Really just to make it clear that the script is not frozen when it is waiting on pods to deploy. If an image must be pulled from an online repository, it can take a while